charts/mattermost-team-edition/Chart.yaml

```diff
@@ -1,8 +1,9 @@
-apiVersion: v1
+apiVersion: v2
 description: Mattermost Team Edition server.
+type: application
 name: mattermost-team-edition
-version: 3.22.0
-appVersion: 5.29.0
+version: 4.0.0
+appVersion: 5.32.1
```

Comment:
Sorry I'm a little late but I just installed 4.0.0 and had no problems, but I noticed there's a tag value set to 5.29.0, not this appVersion. If the tag is updated to match this newer app version, this issue comes up https://github.com/mattermost/mattermost-helm/issues/203